tests/integration/test_searchsorted.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+
+def compare_assert(a_np, a_num):
+    if not num.allclose(a_np, a_num):
+        print("numpy, shape " + str(a_np.shape) + ":")
+        print(a_np)
+        print("cuNumeric, shape " + str(a_num.shape) + ":")
+        print(a_num)
+        assert False
+
+
+def generate_random(shape, datatype):
```

Comment:
you call this function only with 1-tuples. can't you just pass volumes directly? if this is copied from some other test you wrote, consider making it a common utility. (at some point we will consolidate all these ad-hoc input generations into one module and make all tests use it.)